src/routes/(app)/admin[[hash=admin_hash]]/+page.svelte

```diff
@@ -89,27 +76,18 @@
 	<h1 class="text-xl">Documentation</h1>
 
 	<p>Select your language :</p>
-	<div class="flex flex-row justify-evenly">
-		<button
-			on:click={() => {
-				lang = 'en';
-				fetchFiles();
-			}}>🇬🇧</button
-		>
-		<button
-			on:click={() => {
-				lang = 'fr';
-				fetchFiles();
-			}}>🇫🇷</button
-		>
+	<form class="flex flex-row justify-evenly" method="GET">
+		<input type="hidden" bind:value={lang} name="lang" />
+		<button on:click={() => (lang = 'en')}>🇬🇧</button>
+		<button on:click={() => (lang = 'fr')}>🇫🇷</button>
```

Comment:
here you can do `button` with individual `formAction` 

Or just links styled as buttons, outside a form

Setting the lang var and then submitting immediately doesn't seem 100% stable in different browsers, ...